Optionally set refuse_seconds filter in ACCEPT and DECLINE requests

papagian opened the pull request.

Commit:
Optionally set refuse_seconds in ACCEPT and DECLINE requests

osallou commented on mesoshttp/offers.py:
```diff
@@ -25,7 +25,7 @@ def get_offer(self):
         return self.offer
 
-    def accept(self, operations):
+    def accept(self, operations, refuse_seconds=None):
```
to be more generic, for more upcoming options, would be better to get something like:

    def accept(self, operations, options=None):
    ... and take refuse_seconds from options['filters']
        if options and options.get('filters'):
           message['accept']['filters'] = options.get('filters')

Commit:
Support various options in ACCEPT and DECLINE requests

Substitute refuse_seconds parameter in ACCEPT and
DECLINE requests with a generic one.
